views/pulsar/v2/forms.html.twig

```diff
@@ -0,0 +1,204 @@
+{% extends "form_div_layout.html.twig" %}
+
+{% block form_widget_simple %}
+    {% set attr = attr|merge({'class': (attr.class|default('') ~ ' form__control')|trim}) %}
+    {{ parent() }}
+{% endblock form_widget_simple %}
+
+{# Remove the surrounding div #}
+{% block form_widget_compound %}
+    {%- if form is rootform -%}
```

Comment:
Indeed. The hyphen in `{%-` trims whitespace. I think we should be doing that and follow what Symfony are doing upstream instead of using `spaceless` - https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig